.github/actions/test_action/action.yml

```diff
@@ -0,0 +1,6 @@
+name: 'CI_action'
+description: 'Runs unit tests upon push.'
+runs:
+  using: 	'docker'
+  image: 	'./../../../Dockerfile'
+  entrypoint:	"/home/jovyan/run_tests.sh"
```

Comment:
who is this user?

Reply:
jovyan is the user that is created in the jupyter/base-notebook Dockerfile. jovyan/work is the WORKDIR where our Dockerfile installs u8timeseries.